File: Command-R+/Pythom/2-3.py
input_file = "input.txt"
output_file = "output.txt"

try:
    with open(input_file, 'r') as infile, open(output_file, 'w') as outfile:
        content = infile.read()
        outfile.write(content)
    print("File read and write operation completed successfully.")
except FileNotFoundError:
    print("Error: One or both files not found.")

File: Command-R+/C++/2-3.c
#include <iostream>
#include <fstream>

int main() {
    std::ifstream inputFile("input.txt");
    std::ofstream outputFile("output.txt");

    if (inputFile.is_open() && outputFile.is_open()) {
        std::string line;
        while (getline(inputFile, line)) {
            outputFile << line << std::endl;
        }
        std::cout << "File read and write operation completed successfully." << std::endl;
    } else {
        std::cout << "Error opening files." << std::endl;
    }

    inputFile.close();
    outputFile.close();

    return 0;
}

